### DISCUSSION TITLE
Nested pages (collections)

### DISCUSSION THREAD

**Discussion by stijns96** [directus/directus#24071] [Feature Requests]
### Summary

I'm coming from Storyblok and I'm currently testing out Directus. One of the things I'm missing here (or I don't know how to do it), are nested pages (collections).

In Storyblok you can simply create folders with a root page and sub pages. This automatically follows the url structure.

I think that this is quite a common thing to do with a headless CMS, and with this feature, you don't really need a developer to create these pages in your code base as well. Simply use `[...slug].vue` (in Nuxt), and you're good to go.

### Motivation

A more flexible approach to setup nested pages without the need of extra dev work

### Detailed Design

Not really sure about this, but I would be great of we could create folders inside collections as well. Maybe by adding a "Allow folders" checkbox to the settings. There should also be some logic that these pages (collections) are following the url (id) structure, so we can dynamically fetch them with framework features.

### Must Have:
- A Collection folders
- B URL (ID) aware nested collections

### Drawbacks

-

### Alternatives

-

### Adoption Strategy

-

### Unresolved Questions

-

**Comment by rijkvanzanten** [comment]
Hoi @stijns96! While I'm not intimately familiar with Storyblok's approach to page handling, it sounds like what you're looking for is a one-to-many relationship from your pages to collection back to your pages collection. That way every page has a "parent page" and "children pages". 

I'll close this for now as I believe this is more a usage question than a new feature request. Happy to keep chatting here or on [Discord](https://directus.chat) 🙂
